src/payment/strategies/braintree/braintree-payment-processor.ts

```diff
@@ -150,6 +150,9 @@ export default class BraintreePaymentProcessor {
                 amount,
                 nonce,
                 removeFrame,
+                onLookupComplete: (_data, next) => {
+                    next();
```

Comment:
Do we need to inspect [data](https://braintree.github.io/braintree-web/current/ThreeDSecure.html#~verificationData) parameter before calling `next`? i.e.: Is there a scenario where we shouldn't call `next` and have to cancel the verification flow (i.e.: if the lookup fails or authentication is not required)?

Reply:
Right now i cant see any real cases. But according to this reference
https://braintree.github.io/braintree-web/current/ThreeDSecure.html#cancelVerifyCard its possible to use `cancelVerifyCard` callback type if need.

Reply:
Ok if you can’t think of a real case right now, then I’m happy with what you have got. We can change it later if we discover there’s a real need.